other/ethhub-cftc-response.md

```diff
@@ -24,6 +24,8 @@ The functionality of the underlying asset Ether is actually very similar to Bitc
 
 The simplest use case of the Ethereum network is value transfer by sending Ether \(which has an attached value\) from one person to another. However, the amount of applications being built on top of the program is growing at a rapid pace. Currently, the most active applications are those related to "Decentralized Finance". As of 12/11/18, there is currently $120,000,000 worth of Ether being used in Decentralized Finance applications.
 
+There are also a number of "tokenized" of assets commonly referred to as "tokens" on the network. These tokens are smart contracts that contain a balance sheet and transaction log of users swapping tokens between each other and or the smart contract itself. Before Ethereum was created a new blockchain with its own token would have to be created, this created a walled garden where blockchains could not interact with each other nor could users easily swap coins/tokens easily without the use of a cryptocurrency exchange. However with Ethereum a two token smart contracts can be accessed and used by a user easily without an exchange as the base network provides the link between the two and allows transactions to proceed without an exchange. This capability was first implemented in Ethereum and remains a key feature of the network.
```

Comment:
I like the addition of explaining tokens but I don't think it is quite accurate to say that blockchains couldn't interact with one another. I don't think we need to explain DEXs here. "This capability was first implemented in Ethereum and remains a key feature of the network" is misleading because it might imply that swaps were part of the core protocol.